src/dynamodb/gameStateDBService.ts

```diff
@@ -151,6 +151,29 @@ export const getInteractionCount = async (gameId: string): Promise<number | unde
 /* ----------------------------------------------------------------------------
  * Update
  * ------------------------------------------------------------------------- */
+
+/*
+ * Add new game state or replace existing game state with the same game id into GAME_STATE_TABLE.
+ * Mainly to be used to easily add game states for testing purposes.
+ * FOR USE IN TESTING ONLY.
+ */
+export const testReplaceGameState = async (newGameState: GameState): Promise<GameState | undefined> => {
+  const putItemParm: DocumentClient.PutItemInput = {
+    TableName: GAME_STATE_TABLE,
+    Item: newGameState,
+    ReturnValues: 'ALL_OLD',
+  };
+  const getItemParm: DocumentClient.GetItemInput = {
+    TableName: GAME_STATE_TABLE,
+    Key: {
+      gameId: newGameState.gameId,
+    },
+  };
+  await DB.put(putItemParm).promise();
+  const gs = await DB.get(getItemParm).promise();
+  return parseDynamoDBItem<GameState>(gs);
```

Comment:
maybe use:
```ts
return await getGameStateByGameId(newGameState .gameId)
```

Reply:
fixed, apparently the await keyword is redundant according to eslint, so i've removed it